DXMainClient/Online/CnCNetManager.cs

```diff
@@ -28,11 +28,13 @@ public class CnCNetManager : IConnectionManager
         public delegate void UserListDelegate(string channelName, string[] userNames);
 
         public event EventHandler<ServerMessageEventArgs> WelcomeMessageReceived;
+        public event EventHandler ChannelMOTDComplete;
```

Comment:
What is it short for?

Reply:
> What is "MOTD" short for?

Message of the day

Reply:
Use "MessageOfTheDay"/"MsgOfTheDay" to replace "MOTD" in the method name or at least leave comments on this abbreviation